### ISSUE TITLE
[Color Picker] Provide a label for the component's RGB transparency input (CAL-AMP49) 

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#13540]
**monday.com sync:** #10821962093

### Violation: 
 Provide a valid label for form fields 

### Recommended solution
Add a label to the component's RGB transparency input.

<img width="259" height="360" alt="Image" src="https://github.com/user-attachments/assets/0dd4bbd2-e475-4d52-8bb9-cd06add4a5cd" />

### WCAG Reference: 
- [2.5.3 Label in Name](https://www.w3.org/WAI/WCAG22/Understanding/label-in-name) 

### Severity: 
 Critical 

### Media Type: 
 ARIA 

--- 
### Areas for Remediation: 
- Title: CAL-AMP49 Provide a valid label for form fields 
- Module: M7 Table and color picker 
- Finding ID: [CAL-AMP49](https://esri.hub.essentia11y.com/short-link/49iYXqu1uBIdh-AE) 

**Issue**
There is an "Opacity" input field without a programmatically associated label describing its purpose.

**User Impact**
Screen reader users and users with cognitive disabilities may be unable to determine what these controls are for. Speech input users will have difficulty navigating to them.

**Code Reference** 
 ```
<input type="text" aria-errormessage="inputNumberValidationMessage" aria-invalid="false" aria-label="" enterkeyhint="" inputmode="decimal" placeholder="">

```

**Steps to Reproduce**
1. Locate the form field.
2. Inspect it with Chrome DevTools.
3. In the Accessibility tab, expand the Computed Properties section.
4. Review the value for "Name".
 
**Recommendation**
Provide a valid label for form fields.

Labels must meet the following requirements:
- The label must be visible.
- The label must be persistent. That is, a label must remain visible when text is entered into the field or an option is selected.
- The label must be programmatically associated with the form field. The most common way to do this is with a ```<label>``` element with a for attribute set to the ID of the field.
- The label must provide the accessible name of the field, or the label's exact text must be included in the accessible name.

**Compliant Code Example**
```
<!--Example using label element--> 
<label for="amount ">Amount:</label> <input id="Amount" type="text" name="amount" size="30"> 
---
<!--Example label has no "for" attribute but does wrap the input element)--> 
<label>Amount: <input id="amount " type="text" name="amount " size="30"></label> 
---
<!--Example using aria-labelledby--> 
<label id="lblAmount ">Amount:</label> <input aria-labelledby="lblAmount" id="Amount" type="text" name="amount" size="30"> 
---
<!-- Example using aria-label attribute - visual label exists but is not associated--> 
<span>Amount:</span> 
<input aria-label="Amount:" type="text" id="amount" size="30"> 
---
<!--Example using title attribute - visual label exists but is not associated--> 
<span>Amount:</span> 
<input title="Amount:" type="text" id="amount" size="30">
```

 **Issue Image** 
 https://desktopqa/aic_lap/Calcite/Calcite2025/CAL-AMP49.png 
 

 --- 
### Additional Resources: 
- [LA Platform Report Link](https://esri.hub.essentia11y.com/short-link/49iYXqu1uBIdh-AE) 
- [Esri Accessibility Resources](https://esriis.sharepoint.com/sites/Accessibility)
- Report Source: Product accessibility evaluation conducted on Calcite by Level Access and delivered August, 2025.

**Comment by geospatialem** [comment]
In testing the above context is provided to assistive technologies, so the above is invalid. Closing the above out as unplanned.

cc @jcfranco @Elijbet
